GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -95,6 +75,18 @@ - (nullable NSString *)openIDRealm {
   return grantedScopes;
 }
 
+- (GIDConfiguration *)configuration {
+  static GIDConfiguration *configuration;
```

Comment:
In this context, static variables are going to be shared across method calls and all instances of GIDGoogleUser, so you'll probably want to go with instance vars here.

Reply:
Changed static to __block.